src/requirements-dev.txt

```diff
@@ -0,0 +1,14 @@
+-r requirements.txt
+
+pytest
+pytest-asyncio                        
+pytest-cov                            
+pytest-flakefinder                     
```

Comment:
As far as I'm aware, we are not using `pytest-flakefinder` since it conflicts with `pytest-asyncio`.

Reply:
fixed